package.json

```diff
@@ -2,10 +2,19 @@
   "name": "ember-exam",
   "version": "0.7.1",
   "description": "Run your tests with randomization, splitting, and parallelization for beautiful tests.",
+  "keywords": [
```

Comment:
These restructures came from default ember-cli blueprint - my intention was to use make future upgrades easier but I can revert the properties ordering if these changes do not make sense